addons/l10n_uy/data/template/account.tax-uy.csv

```diff
@@ -20,6 +20,6 @@
 "","","","","","","","","","base","refund","-Tax Base Purchases","","",""
 "","","","","","","","","","tax","refund","-VAT Purchases - paid","uy_code_11501","",""
 "vat6","0% EXEMPT","Purchases Exempt from VAT","Purchases Exempt from VAT","vat","0.0","fixed","purchase","tax_group_exenton","base","invoice","+Base Purchases 0%","","Compras Exento IVA","Compras Exentos IVA"
-"","","","","","","","","","tax","invoice","","uy_code_21403","",""
+"","","","","","","","","","tax","invoice","","uy_code_11503","",""
 "","","","","","","","","","base","refund","-Tax Base Purchases","","",""
-"","","","","","","","","","tax","refund","","uy_code_21403","",""
+"","","","","","","","","","tax","refund","","uy_code_11503","",""
```

Comment:
Hi @JulienVR 
Thank you for the review.

We notice the problem of this account because we have in our side a constraint that do not let to add non valid accounts types to a tax because can affect to the tax reports.

Also, as it is right now we don't have consistency, Becase we have a liability_payable account configured in on the tax, but if you check the domain in the tax view it explicitly said that we can not have use a liability_payable account type 
![image](https://github.com/user-attachments/assets/5679f59b-ea25-4c94-83cb-babd7fb077f3)

Reply:
Hi @JulienVR 
Please let me know if you have any question about this or if there is anything needed to continue with the merge of this fix.

Best Regards

Reply:
Hello @zaoral 

For this PR, I was checking but in the case of purchases I see that the tax is fixed, shouldn't it be percentage? Account is ok. Now, in sales, I also see it as fixed, but also the account being used is still payable... What I understood in the description of the PR is that it shouldn't be payable right? We're using 214030 Tax to be Paid, and it is a payable.

Let me know please.

Fernando

@ren-odoo @JulienVR

Reply:
HI @fvz-odoo

Thank you for your feedback

Tax is 0 percent, if you like we can change it to percent. Not sure why we have it as fixed. Do you want me to make this change in 17.0 estable version, or do that fix on master?

About the Tax to be Paid on sales, you are right, not sure why did not get us any error, You want me to create a new account for VAT Exempt Sales? I will wait your confirmation

Best Regards

Reply:
Hello @zaoral 

Thanks for your answer. for the tax fixed or percent, the best person to answer if we can change it in stable is @JulienVR, let us know what you think. 

For the second question, what's the easiest, create a new account with the correct configuration or fix the existing? In my opinion I would suggest to fix the existing, but not sure if this can also be done in stable, @JulienVR what do you think?

Thanks team!

Cheers!

Fernando

@ren-odoo

Reply:
Hi @fvz-odoo and @JulienVR 

I have fix it the accounts: Now both 0% taxes (sale and purchase) has the proper account, and the taxes config was also changed from fixed to percent

I think is ready for your review
Best Regards